action/kafkaFeedWeb.js

```diff
@@ -61,6 +61,35 @@ function main(params) {
 
                     resolve(common.webResponse(statusCode, body));
                 });
+        } else if (params.__ow_method === "get") {
```

Comment:
Code is the same as in messageHubFeedWeb.js. Should make it a shared method between the two.

Reply:
you'll notice the code between messageHubFeedWeb.js and kafkaFeedWeb.js is entirely duplicated. i think it's definitely fair to get rid of the duplication. but i don't this this is the the place to do it. it should be done under a different issue/epic

Reply:
Can you open an issue for this?

Reply:
https://github.com/apache/incubator-openwhisk-package-kafka/issues/218